test/system/500-networking.bats

```diff
@@ -589,4 +589,22 @@ load helpers
     run_podman network rm -t 0 -f $netname
 }
 
+@test "podman run CONTAINERS_CONF dns options" {
+    # Test on the CLI and via containers.conf
+    containersconf=$PODMAN_TMPDIR/containers.conf
+
+    cat >$containersconf <<EOF
+[engine]
+  dns_searches  = [ "example.com"]
+  dns_servers = [
+    "1.1.1.1",
+    "1.0.0.1",
+    "8.8.8.8",
+]
+EOF
+    run_podman run --rm $IMAGE grep "example.com" /etc/resolv.conf
```

Comment:
Did you mean to prefix with `env CONTAINERS_CONF=$containersconf` ?

Reply:
Yup, strange that this worked when I was testing. I must have screwed up tests.